atorch/examples/llama2/README.md

```diff
@@ -402,4 +402,25 @@ pip install -r requirements.txt
 # e.x. in a 8-gpu system, to run mp-2 dp-2 pp-2, 
 # use `PIPELINE_PARALLEL_SIZE=2 MODEL_PARALLEL_SIZE=2 sh ds_3d_llama2_entry.sh`
 sh ds_3d_llama2_entry.sh
-```
\ No newline at end of file
+```
+
+### Performance
+- Model: Llama2-7b
+- Cluster: 8 A100-80GB GPUs in each node, with NVLink and RDMA
+
+| Method | Use LoRA | Num of GPUs | TP | PP | DP | Batch Per GPU | Grad Accu Steps | Global Batch Size | TFLOPs | Mem Max allocated(MB) |
```

Comment:
lora flops computation seems not correct.
Also for lora training, add "wrap_trainable_outmost": True, in fsdp config would increase perf. In my 8-A100 test, the printed flops increase from 196 to 239.